docs/reference/modules/network/threading.asciidoc

```diff
@@ -0,0 +1,106 @@
+[[modules-network-threading-model]]
+==== Networking threading model
+
+This section describes some technical details about the threading model used by
+the networking subsystem in {es}, which may be important for advanced users to
+diagnose certain problems in a cluster.
```

Comment:
```suggestion
This section describes the threading model used by the networking subsystem in
{es}. While this information isn't required to use {es}, it may be useful to advanced
users who are diagnosing network problems in a cluster.
```